de4a-kafka-client/src/main/java/eu/de4a/kafkaclient/DE4AKafkaSettings.java

```diff
@@ -38,6 +38,7 @@
   private static final AtomicBoolean s_aLoggingEnabled = new AtomicBoolean (true);
```

Comment:
In the meantime I would even use all UPPERCASE names for all the static final things